### PR TITLE
OCD-3208: Add api tests to validate version merge permission

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: chpl/chpl-api/e2e/script/newman-run-dev.js [hunk 1]
```diff
@@ -80,6 +90,7 @@ const jobs = [
     cb => newman.run(developerControllerDataTests,cb),
     cb => newman.run(developerControllerNoDataTests,cb),
     cb => newman.run(productsControllerTests,cb),
+	cb => newman.run(productVersionControllerTests,cb),
```
Comment (palakpatel311) [inline chpl/chpl-api/e2e/script/newman-run-dev.js]: This should be 4 spaces and not tab as per our formatter. Same with other scripts

File: chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,144 @@
+{
+	"info": {
+		"name": "product-version-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "PUT /versions - merge request permission restriction validation - anonymous",
```
Comment (palakpatel311) [inline chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json:8]: I don't see these test running in bamboo. Is it pushed to dev?

Comment (ashwinimore) [reply chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json:8]: nope it is not yet pushed to DEV, I wanted to check out the tests from other PR that I pushed yesterday. I am going to push these out today

File: chpl/chpl-api/e2e/env/sample.postman_environment.json [hunk 3]
```diff
@@ -11,6 +11,26 @@
             "key": "apiKey",
             "value": "enterKeyHere",
             "enabled": true
+        },
+		{
+            "key": "authurl",
+            "value": "https://chpl.ahrqdev.org/rest/auth/authenticate",
```
Comment (palakpatel311) [inline chpl/chpl-api/e2e/env/sample.postman_environment.json]: We may want to have this url as generic url name

**Review by andlar — CHANGES_REQUESTED** [review]
Why are there 300 files in this PR, and a huge chunk of them are functional code? This is an AQA test PR, right? It shouldn't have code changes in the functional source stuff

**Comment by ashwinimore** [comment]
> 
> 
> Why are there 300 files in this PR, and a huge chunk of them are functional code? This is an AQA test PR, right? It shouldn't have code changes in the functional source stuff

that's a mishap happened on my end couple of hours ago when I pushed a commit after I merged the branch into development. I was hoping to get your help tomorrow morning to 'fix' this, if possible. I wanted to try reset but didn't know if that'd do it

File: chpl/chpl-api/e2e/env/sample.postman_environment.json (hunk 3 above)
Comment (ashwinimore) [reply chpl/chpl-api/e2e/env/sample.postman_environment.json]: Pushed a commit for this

File: chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json (hunk 2 above)
Comment (ashwinimore) [reply chpl/chpl-api/e2e/collections/product-version-controller.postman_collection.json:8]: Tests were pushed to dev, plan was run and tests ran as expected

**Review by andlar — APPROVED** [review]

**Review by palakpatel311 — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Pull request merged**